lib/shoryuken/util.rb

```diff
@@ -4,13 +4,13 @@ def logger
       Shoryuken.logger
     end
 
-    def fire_event(event, reverse = false)
+    def fire_event(event, options = {}, *event_args)
```

Comment:
The latter `event_options` is only for making it more explicit, like `fire_event(:dispatch, false, queue_name: queue.name)`